backend/gateway/gateway.go

```diff
@@ -39,7 +41,30 @@ type ComponentFactory struct {
 	Modules    module.Factory
 }
 
+func loadEnv(f *Flags) {
+	// Order is important as godotenv will NOT overwrite existing environment variables.
+	envFiles := f.EnvFiles
+
+	for _, filename := range envFiles {
+		// Use a temporary logger to parse the environment files
+		tmpLogger := newTmpLogger().With(zap.String("file", filename))
+
+		p, err := filepath.Abs(filename)
+		if err != nil {
+			tmpLogger.Fatal("parsing .env file failed", zap.Error(err))
+		}
+		// Ignore lint below as it is ok to to ignore dotenv loads as not all env files are guaranteed
+		// to be present.
+		// nolint
+		err = godotenv.Load(p)
+		if err == nil {
+			tmpLogger.Info("successfully loaded environment variables")
```

Comment:
add which file was successful in this log as there can be many

Reply:
It's part of the [logger](https://github.com/lyft/clutch/pull/1761/files/c14ffdfbf3e33a1f644c3fda4328302e37faf62d#diff-df82dea03f5bfbf3e5b85c91924864bbc4c9d331eef1b39a6d00f2fae71cbc76R50)